### PR TITLE
Added link to inclusivity document

### PR DESCRIPTION
**Pull request by janeman98**
Added link from contributor guide to inclusivity document.

Fixes # 454

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1632)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Fixes https://github.com/kubeflow/website/issues/454

File: content/docs/about/contributing.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ again.
 ### Follow the code of conduct
 
 Please make sure to read and observe our [Code of Conduct](https://github.com/kubeflow/community/blob/master/CODE_OF_CONDUCT.md).
+If you are a Kubeflow leader, please read our [inclusivity document](https://github.com/kubeflow/community/blob/master/INCLUSIVITY.md).
```
Comment (sarahmaddox) [inline content/docs/about/contributing.md]: I'm thinking the inclusivity guide is relevant to all community members. How about saying "... our Code of Conduct and inclusivity document."

Comment (janeman98) [reply content/docs/about/contributing.md]: Thanks @sarahmaddox !
Sound good. I have modified the file. Please review/approve. Let me know if further change is needed.

Comment (sarahmaddox) [reply content/docs/about/contributing.md]: Thanks!

**Pull request merged**